cfgov/data_research/tests/test_forms.py

```diff
@@ -10,22 +10,26 @@
 class ConferenceRegistrationFormTests(TestCase):
     capacity = 100
     govdelivery_code = 'TEST-CODE'
+    govdelivery_question_id = 12345
```

Comment:
Minor thing that doesn't affect the execution of these tests: this (and other references below, see diff [here](https://gist.github.com/chosak/ef5cee23b6f56316357ed472af5bbe56)) should probably be a string `'12345'` to be consistent with how we expect the data to be passed in. This doesn't break the tests because [this line](https://github.com/cfpb/cfgov-refresh/pull/4519/files#diff-17c90d3c3fe8d73df852c60805e01d64R174) is mocked out, but in practice this will always need to be a string to avoid an error [here](https://github.com/cfpb/govdelivery/blob/master/govdelivery/api.py#L125).

Reply:
Thanks, updating that.